### DISCUSSION TITLE
Change order that user fields appear on signup page?

### DISCUSSION THREAD

**Discussion by padraigpearlman** [fief-dev/fief#135] [Q&A]
I have a user field for the user's full name. This is what the signup page looks like right now:

<img width="443" alt="Screenshot 2023-01-21 at 8 08 59 PM" src="https://user-images.githubusercontent.com/109485814/213896043-ba26356e-2c65-48b1-af9c-447ea3b1bbb2.png">

Is there a way to change the order so that "Full name" appears first?

**Comment by frankie567** [comment]
Hey 👋  Unfortunately, no, that's not possible at the moment.

I'll add this to the backlog :)
